src/main/java/seedu/jarvis/logic/commands/delete/DeleteConsultationCommand.java

```diff
@@ -1,12 +1,26 @@
 package seedu.jarvis.logic.commands.delete;
```

Comment:
Great work to abstract away the Delete commands 🥇 It is definitely not an easy task.